src/Tizen.NUI/src/public/Animation/Animatable.cs

```diff
@@ -199,6 +199,18 @@ public PropertyValue GetProperty(int index)
             return ret;
         }
 
+        /// <summary>
+        /// Retrieves a property value.
```

Comment:
```suggestion
        /// Retrieves the latest rendered frame value of the property.
```

I think we'd better match the comment with dali-core side comments. Actually, should be difference with `GetProperty` API comments.

https://github.com/dalihub/dali-core/blob/master/dali/public-api/object/handle.h#L480

Reply:
Oh. good good. I fixed.